src/AI/AIUtils.java

```diff
@@ -214,6 +216,36 @@ public static CommanderAbility queueCromulentAbility(Queue<GameAction> q, Comman
     }
     return retVal;
   }
+  
+  /**
+   * @return The area threatened by the unit, against the specified target type
```

Comment:
This returns not just area, but also potential damage at each location.

Reply:
Sure. It's up to the caller to determine what "threat" means.